clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureBaseClient.groovy

```diff
@@ -46,4 +57,53 @@ public abstract class AzureBaseClient {
       throw new RuntimeException("Failed to create Azure Resource Management Service", e)
     }
   }
+
+  public String getAzureRESTJson(AzureCredentials creds, String baseUrl, String targetUrl, List<String> queryParameters){
+    String result = null
+    String url = AzureUtilities.getAzureRESTUrl(subscriptionId, baseUrl, targetUrl, queryParameters)
+
+    try {
+      // create HTTP Client
+      def config = buildConfiguration(creds)
+      HttpClientBuilder httpClientBuilder = config.create(HttpClientBuilder.class)
+
+      String proxyHost = System.getProperty("http.proxyHost");
```

Comment:
`System.getProperty` doesn't seem quite right. 

Are these not properties that would otherwise end up in a yml config block?